### ISSUE TITLE
This issue

### ISSUE THREAD

**Issue by mdev88** [ecdpalma/parse-save-eventually-issue#1]
Hello. I know you created this years ago, but I am running my own Parse Server instance, and I am having this exact issue.
I was wondering if you had any solution or more information about what caused it.

Thanks!

**Comment by ecdpalma** [comment]
No, I didn't. I stopped using saveEventually and implemented my own queue. Today I would use yigit's job queue along with Android's JobScheduler.

If I remember well, the problem is that the Parse SDK code does *only* the four requests (1 + 3 retries). It cannot tell whether the network is fine or not (firewalled, in the case of this test repo). If network is lost (e.g. disabled), then it receives a broadcast message when it gets recovered (enabled), and then reschedules the requests.

This had been opened in Facebook's bug tracker, but I think it was closed with "by design" or something like that. A proper solution would be to change the SDK code so that saveEventually (or saveInBackground) allows some sort of retry strategy configuration.
